src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -70,17 +70,24 @@ public ApplicationExtract(ILogger<ApplicationExtract> log, ApplyDataContext appl
 
                 foreach (var applicationId in applications)
                 {
-                    var answers = await ExtractAnswersForApplication(applicationId);
+                    try
+                    {
+                        var answers = await ExtractAnswersForApplication(applicationId);
 
-                    using (var transaction = _applyDataContext.Database.BeginTransaction())
+                        using (var transaction = _applyDataContext.Database.BeginTransaction())
+                        {
+                            await SaveExtractedAnswersForApplication(applicationId, answers);
+                            await SaveSectorDetailsForApplication(applicationId, answers);
+                            await LoadOrganisationManagementForApplication(applicationId, answers);
+                            await LoadOrganisationPersonnelForApplication(applicationId, answers);
+                            await transaction.CommitAsync();
+                        }
+                        await EnqueueApplyFilesForExtract(applyFileExtractQueue, answers);
+                    }
+                    catch (Exception ex)
                     {
-                        await SaveExtractedAnswersForApplication(applicationId, answers);
-                        await SaveSectorDetailsForApplication(applicationId, answers);
-                        await LoadOrganisationManagementForApplication(applicationId, answers);
-                        await LoadOrganisationPersonnelForApplication(applicationId, answers);
-                        await transaction.CommitAsync();
+                        _logger.LogError(ex, $"Error while processing the ApplicationExtract for application {applicationId} at: {DateTime.Now}");
```

Comment:
Always use structured logging pattern 
`_logger.LogError(ex, "Error while processing the ApplicationExtract for application {applicationId}", applicationId);`
Not need to include time.